lib/registries/addon/components/registries-header/template.hbs

```diff
@@ -40,6 +39,14 @@
                         {{fa-icon 'question'}}
                     </Button>
                 {{/if}}
+                <Button
+                    aria-label={{t 'registries.header.search_button'}}
+                    local-class='SearchIcon'
+                    @layout='fake-link'
+                    {{on 'click' (action this.onSubmit)}}
```

Comment:
drive-by: with Octane, the `action` helper is no longer needed. `this.onSubmit` is bound to the backing class by the `@action` decorator.